### ISSUE TITLE
Twisted 2d visualization

### ISSUE THREAD

**Issue by justinmc** [Autodesk/notebook-molecular-visualization#8]
The 2d visualization can become twisted into configurations that look correct at first glance, but are actually reversed.

In these screenshots, the right right half of bipyridine is actually upside down.  Notice how the selected atom in the 2d visualization corresponds to the selected atom in the 3d visualization in the first screenshot, but in the second one it's on the wrong side:
<img width="892" alt="screen shot 2016-08-01 at 1 34 23 pm" src="https://cloud.githubusercontent.com/assets/389558/17308044/d1628fa6-57ec-11e6-836f-dc13c9d81432.png">
<img width="916" alt="screen shot 2016-08-01 at 1 34 31 pm" src="https://cloud.githubusercontent.com/assets/389558/17308045/d166bc5c-57ec-11e6-9de9-1aef9e835912.png">

**Comment by avirshup** [comment]
Yeah, that's no good! D3 v4.0 may have ways to address this - it's supposed to be much better at finding the globally optimal graph layout

**Comment by justinmc** [comment]
Cool, I did a little work on upgrading to 4.0, we'll talk tomorrow.

**Comment by justinmc** [comment]
@avirshup This is what happens every time I refresh the page with bipyridine in d3 v4 right now.  Should be fixed with that PR if you agree (https://github.com/Autodesk/nbmolviz2d/pull/3).

![d3v4](https://cloud.githubusercontent.com/assets/389558/17491800/ffaeaeb4-5d5d-11e6-91ad-dafdf7c1b643.gif)
